src/webui/index.js

```diff
@@ -183,17 +187,34 @@ module.exports = async function () {
   })
 
   const launchWebUI = ctx.getFn('launchWebUI')
+  const splashScreenPromise = ctx.getProp('splashScreen')
+  if (store.get(CONFIG_KEY) && process.env.NODE_ENV !== 'test') {
```

Comment:
showing the splash screen during testing causing some odd issues with playwright electron and not being able to get the actual webui window.